.github/workflows/issue-comment-created.yml

```diff
@@ -0,0 +1,16 @@
+name: Issue Comment Created Triage
```

Comment:
This will just remove `stale` and `waiting-reply` if a maintainer adds them at creation time, right? Is it possible a maintainer would do that on first file intentionally / do we expect it to happen unintentionally?

Reply:
i think it's when a comment is created on the issue that we'll remove `stale` and `waiting-reply`, does that answer your question?

Reply:
Oh, yup, that makes more sense!